src/index.ts

```diff
@@ -12,43 +22,46 @@ function errorHandler(error: any, request: FastifyRequest, reply: FastifyReply<S
     request.log.error({ ...error, statusCode })
   }
 
-  const response = isClientError
-    ? {
-        error: error.type,
-        message: error.message,
-        statusCode,
-        details: error.validation || error.details,
-      }
-    : {
-        error: error.type,
-        message: 'Something went wrong. Try again later.',
-        statusCode,
-      }
+  let response: any
+  if (isInterference(error) && isClientError) {
+    response = {
+      error: error.type,
+      message: error.message,
+      details: error.details,
+      statusCode,
+    }
+  } else if (error.validation) {
+    response = {
+      error: 'INVALID_REQUEST',
```

Comment:
It is better to put something like `VALIDATION_ERROR` since we have other cases where a request could be not valid.